tests/test_remotesparkmagics.py

```diff
@@ -0,0 +1,94 @@
+from nose.tools import raises, with_setup, assert_equals
+from remotespark.RemoteSparkMagics import RemoteSparkMagics
+from mock import MagicMock
+
+ip = get_ipython()
+magic = None
+
+def _setup():
+	global magic
+	magic = RemoteSparkMagics(shell=ip)
+	ip.register_magics(magic)
+
+def _teardown():
+	pass
+
+@with_setup(_setup, _teardown)
+def test_info_command():
+	mock_method = MagicMock()
+	magic._print_info = mock_method
+	command = "info"
+
+	ip.run_line_magic("spark", command)
+
+	mock_method.assert_called_with()
+
+@with_setup(_setup, _teardown)
+def test_add_endpoint_command():
+	mock_method = MagicMock()
+	magic.add_endpoint = mock_method
+	command = "add"
+	name = "name"
+	language = "python"
+	connection_string = "url=http://location:port;username=name;password=word"
+	line = " ".join([command, name, language, connection_string])
+
+	ip.run_line_magic("spark", line)
+
+	mock_method.assert_called_with(name, language, connection_string)
+
+
+@with_setup(_setup, _teardown)
+def test_delete_endpoint_command():
+	mock_method = MagicMock()
+	magic.delete_endpoint = mock_method
+	command = "delete"
+	name = "name"
+	line = " ".join([command, name])
+
+	ip.run_line_magic("spark", line)
+
+	mock_method.assert_called_with(name)
```

Comment:
The tests I've written so far are just testing the parsing of the commands.  I agree that we need to add tests for the actual functionality.